### PR TITLE
feat(TBD-11152): R2020.9: add support for Databricks 6.4.0

### PR DESCRIPTION
**Pull request by talend-jphautin**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [X] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

TBD-11152

**What is the new behavior?**

add support for databricks 6.4.0

### CODE REVIEW

**Review by AlixMetivier — APPROVED** [review]
lgtm

**Review by CarlosTalend — APPROVED** [review]

**Review by CarlosTalend — APPROVED** [review]

File: main/plugins/org.talend.hadoop.distribution.dbr640/src/main/java/org/talend/hadoop/distribution/dbr640/Dbr640Constant.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+// %InstallDIR%\features\org.talend.rcp.branding.%PRODUCTNAME%\%PRODUCTNAME%license.txt
+//
+// You should have received a copy of the agreement
+// along with this program; if not, write to Talend SA
+// 9 rue Pages 92150 Suresnes, France
+//
+// ============================================================================
+package org.talend.hadoop.distribution.dbr640;
+
+public enum Dbr640Constant {
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.dbr640/src/main/java/org/talend/hadoop/distribution/dbr640/Dbr640Constant.java:15]:
```suggestion
public enum DBR640Constant {
```

Comment (talend-jphautin) [reply main/plugins/org.talend.hadoop.distribution.dbr640/src/main/java/org/talend/hadoop/distribution/dbr640/Dbr640Constant.java]: done

File: main/plugins/org.talend.hadoop.distribution.dbr640/OSGI-INF/dbr640.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<component name="DBR640Distribution">
+    <implementation class="org.talend.hadoop.distribution.dbr640.Dbr640Distribution"/>
+    <service>
+        <provide interface="org.talend.hadoop.distribution.component.HadoopComponent"/>
+        <provide interface="org.talend.hadoop.distribution.component.HiveOnSparkComponent"/>
+        <provide interface="org.talend.hadoop.distribution.component.SparkBatchComponent"/>
+        <provide interface="org.talend.hadoop.distribution.component.SparkStreamingComponent"/>
+    </service>
+
```
[the remark this answers is not in the document]
Comment (talend-jphautin) [reply main/plugins/org.talend.hadoop.distribution.dbr640/OSGI-INF/dbr640.xml]: done

File: main/features/org.talend.hadoop.distribution.feature/feature.xml [hunk 3]
```diff
@@ -24,6 +24,8 @@
   <plugin id="org.talend.hadoop.distribution.custom" download-size="0" install-size="0" version="0.0.0" unpack="false"/>
   <plugin id="org.talend.hadoop.distribution.dbr350" download-size="0" install-size="0" version="0.0.0" unpack="false"/>
   <plugin id="org.talend.hadoop.distribution.dbr550" download-size="0" install-size="0" version="0.0.0" unpack="false"/>
+  <plugin id="org.talend.hadoop.distribution.dbr640" download-size="0" install-size="0" version="0.0.0" unpack="false"/>
+
```
[the remark this answers is not in the document]
Comment (talend-jphautin) [reply main/features/org.talend.hadoop.distribution.feature/feature.xml]: done

File: main/plugins/org.talend.hadoop.distribution.dbr640/.classpath [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con"
```
[the remark this answers is not in the document]
Comment (talend-jphautin) [reply main/plugins/org.talend.hadoop.distribution.dbr640/.classpath:3]: the formatting of this file is eclipse dependent. I do not edit it.

**Comment by talend-jphautin** [comment]
new patch build : https://jenkins-studio.datapwn.com/job/patches/job/patch-manual-build/1814/
new full build : https://jenkins-studio.datapwn.com/job/Studio/job/studio-build/1045/

**Review by CarlosTalend — APPROVED** [review]

**Review by CarlosTalend — APPROVED** [review]

**Pull request merged**